tests/standAloneClient/bpmNotificationTemplatesForBurntSubtitlesAdditionalTriggers.xml

```diff
@@ -0,0 +1,209 @@
+					<item objectType="KalturaEventNotificationParameter">
+						<key>endPoint</key>
+						<description>Server Host</description>
+						<value objectType="KalturaEvalStringField">
+							<code>kConf::get('apphome_url')</code>
+						</value>
+					</item>
+					<item objectType="KalturaEventNotificationParameter">
+						<key>partnerId</key>
+						<description>Partner ID</description>
+						<value objectType="KalturaEvalStringField">
+							<code>$scope->getEvent()->getObject()->getPartnerId()</code>
+						</value>
+					</item>
+					<item objectType="KalturaEventNotificationParameter">
+						<key>adminSecret</key>
+						<description>Partner Administrator Secret</description>
+						<value objectType="KalturaEvalStringField">
+							<code>$scope->getEvent()->getObject()->getPartner()->getAdminSecret()</code>
+						</value>
+					</item>
+					<item objectType="KalturaEventNotificationParameter">
+						<key>entryId</key>
+						<description>Entry ID</description>
+						<value objectType="KalturaEvalStringField">
+							<code>$scope->getEvent()->getObject()->getId()</code>
+						</value>
+					</item>
+				</contentParameters>
+				<eventConditions objectType="array">
+					<item objectType="KalturaEventObjectChangedCondition">
+						<description>Entry replacement status changed</description>
+						<modifiedColumns>replacementStatus</modifiedColumns>
+					</item>
+					<item objectType="KalturaFieldMatchCondition">
+						<description>replacement status equals</description>
+						<field objectType="KalturaEvalStringField">
+							<code>$scope->getObject()->getReplacementStatus()</code>
+						</field>
+						<values objectType="array">
+							<item objectType="KalturaStringValue">
+								<value>{field_trigger_value}</value>
+							</item>
+						</values>						
+					</item>
+				</eventConditions>
+				<userParameters objectType="array">
+					<item objectType="KalturaEventNotificationParameter">
+						<key>field_trigger_value</key>
+						<description>fields value for triggering</description>
+						<value objectType="KalturaStringValue">
+							<description>none</description>
+							<value>0</value>
+						</value>
+					</item>
+				</userParameters>
+			</template>
+		</request>
+		<request service="eventNotificationTemplate" action="updateStatus" plugin="eventNotification" partnerId="0">
+			<id>{3:result:id}</id>
+			<status>1</status><!-- EventNotificationTemplateStatus::DISABLED -->
+		</request>
+	</multirequest>
+	<multirequest continueOnError="1">
+		<!-- Caption-Asset Status Equals -->
+		<request service="eventNotificationTemplate" action="listAction" plugin="eventNotification" partnerId="0">
+			<filter objectType="KalturaEventNotificationTemplateFilter">
+				<typeEqual>{php:return KalturaEventNotificationTemplateType::BPM_START;}</typeEqual>
+				<partnerIdEqual>0</partnerIdEqual>
+				<statusEqual>1</statusEqual>
+				<systemNameEqual>BPM_START_METADATA_FIELD_CHANGED</systemNameEqual>
```

Comment:
This template will only work for entry metadata, so better update its system name to reflect it